docs/component/rl.rst

```diff
@@ -0,0 +1,242 @@
+.. _rl:
+========================================================================
+Reinforcement Learning in Quantitative Trading
+========================================================================
+.. currentmodule:: qlib
+
```

Comment:
Suggest adding a summary upfront to describe what kind of problem we intend to solve.